R/model.R

```diff
@@ -525,28 +524,41 @@ predict.keras.engine.training.Model <- function(object, x, batch_size=NULL, verb
 #' @inheritParams predict.keras.engine.training.Model
 #' 
 #' @param object Keras model object
-#' 
+#' @param steps Total number of steps (batches of samples) before declaring the
+#'   evaluation round finished. The default `NULL` is equal to the number of 
+#'   samples in your dataset divided by the batch size.
+#'   
 #' @details The input samples are processed batch by batch.
 #' 
 #' @family model functions
 #' 
 #' @export
-predict_proba <- function(object, x, batch_size = 32, verbose = 0) {
-  object$predict_proba(
+predict_proba <- function(object, x, batch_size = NULL, verbose = 0, steps = NULL) {
+  args <- list(
     x = keras_array(x),
     batch_size = as.integer(batch_size),
     verbose = as.integer(verbose)
   )
+  
+  if (keras_version() >= "2.1.3")
+    args$steps <- as.integer(steps)
```

Comment:
This needs to be `as_nullable_integer()` to account for a `NULL` values for `steps`